release_notes.md

```diff
@@ -1,3 +1,83 @@
+***********************************
+# Release Notes for OpenMDAO 3.22.0
+
+December 14, 2022
+
+OpenMDAO 3.22.0 contains a variety of new user-facing capabilities and bug fixes.
+Here are some of the highlights:
+
+We've removed our dependency on the tabulate package.
+The dependencies on pyDOE2 and pyparsing are no longer required unless the user
+attempts to use an OpenMDAO feature that requires them. In this case, that means
+the DOEDriver or the external code file-wrapping capabilities, respectively.
+
+Setting `prob.model` to a component is now deprecated. Doing so would break the way in which OpenMDAO automatically adds
```

Comment:
I didn't think the indepvarcomp was the reason, since openmdao doesn't need to add an IVC if there isn't a group. The original github issue was someone using promotes as an argument to the component, which I think is how this all started. I think the best thing to say here is that the tech burden of supporting this corner case outweighed its usefulness.

Might want to also say that prob.model must always be a Group now, just to drive the point home.